Configurations/10-main.conf

```diff
@@ -1449,8 +1452,10 @@ my %targets = (
         multilib         => "-x64",
     },
     "VC-WIN32" => {
-        # x86 Win32 target defaults to ANSI API, if you want UNICODE,
-        # configure with 'perl Configure VC-WIN32 -DUNICODE -D_UNICODE'
+        # x86 Win32 target used to default to ANSI API, but not anymore,
+        # UNICODE is the only option. The original reason for providing
+        # UNICODE as opt-in option was because ANSI API was *native*
+        # system interface for [no longer supported] Windows 9x.
```

Comment:
This comment should be moved to `VC-noCE-common`, where `UNICODE` and `_UNICODE` are defined...

(and quite frankly, the definition of those two should be moved to `VC-common`, and the comment as well, but that might be for another PR)

Reply:
Let's save this for another merge request. I.e. some clean up on defines and commentary.